stock_manual_transfer/views/stock_manual_transfer_views.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="stock_manual_transfer_form_view" model="ir.ui.view">
+        <field name="name">stock.manual_transfer.form</field>
+        <field name="model">stock.manual_transfer</field>
+        <field name="arch" type="xml">
+            <form string="Manual Transfer">
+                <header>
+                    <field name="picking_ids" invisible="1" />
+                    <button
+                        name="action_validate"
+                        type="object"
+                        string="Make transfers"
+                        invisible="state != 'draft'"
+                        class="oe_highlight"
+                        groups="stock_manual_transfer.group_stock_manual_transfer"
+                    />
+                    <button
+                        name="action_view_pickings"
+                        type="object"
+                        string="View Transfers"
+                        invisible="not picking_ids"
+                        class="oe_highlight"
+                        groups="stock_manual_transfer.group_stock_manual_transfer"
+                    />
+                    <field name="state" widget="statusbar" />
+                </header>
+                <sheet>
+                    <group>
+                        <h1>
+                            <field name="name" force_save="1" />
+                        </h1>
+                    </group>
+                    <group col="4">
+                        <field name="warehouse_id" options="{'no_create': True}" readonly="state != 'draft'" />
```

Comment:
Why this?

The state attribute was not here.

If it's due to the Python attribute not being considered anymore, then you should remove that attribute from the field declaration.

Reply:
Done, thanks for the info.